src/cider/examples/ExpressionEvalExample.cpp

```diff
@@ -31,21 +31,18 @@
 
 int main(int argc, char** argv) {
   // Example 1: generate for expression "a * b + b"
-  SubstraitExprBuilder* builder = new SubstraitExprBuilder();
-  ::substrait::NamedStruct* schema =
-      SubstraitExprBuilder::makeNamedStruct(builder,
-                                            {"a", "b"},
-                                            {CREATE_SUBSTRAIT_TYPE_FULL_PTR(I64, false),
-                                             CREATE_SUBSTRAIT_TYPE_FULL_PTR(I64, false)});
-  ::substrait::Expression* field0 = SubstraitExprBuilder::makeFieldReference(0);
-  ::substrait::Expression* field1 = SubstraitExprBuilder::makeFieldReference(1);
-  ::substrait::Expression* multiply_expr = SubstraitExprBuilder::makeExpr(
-      builder, "multiply", {field0, field1}, CREATE_SUBSTRAIT_TYPE_FULL_PTR(I64, false));
-  ::substrait::Expression* add_expr =
-      SubstraitExprBuilder::makeExpr(builder,
-                                     "add",
-                                     {multiply_expr, field1},
-                                     CREATE_SUBSTRAIT_TYPE_FULL_PTR(I64, false));
+  SubstraitExprBuilder builder({"a", "b"},
+                               {CREATE_SUBSTRAIT_TYPE_FULL_PTR(I64, false),
+                                CREATE_SUBSTRAIT_TYPE_FULL_PTR(I64, false)});
+  ::substrait::Expression* field0 = builder.makeFieldReference(0);
```

Comment:
Do we have required version for Substrait? If so, we need to document it.

Reply:
Yeah, I think so. We will need latest substrait after necessary changes finish upstream.